blog/content/introtogonum.md

```diff
@@ -0,0 +1,154 @@
+# The Gonum Numeric Computing Packages
+
+## Gonum -- The Basics
+
+Gonum is a set of packages designed to make writing numeric and scientific
```

Comment:
A little too late: I'm not a native speaker but I've always seen "numerical" being used rather than numeric. Numerical analysis, numerical solution, numerical methods ...